helm/nessie/templates/_helpers.tpl

```diff
@@ -23,6 +23,13 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{/*
+Create a random cache-invalidation token.
+*/}}
+{{- define "nessie.cacheInvalidation.token" -}}
+{{- randAlpha 24 | nospace -}}
```

Comment:
@adutra : What's your take on this? I wonder how it might work when updating config, for example... will it not cause a new token (and a total restart for all nodes) even for minor deployment changes?

Reply:
I wouldn't mind to set this value to a fixed/constant one. For k8s deployments, the token is probably not that useful, given that you cannot really access the management port from anywhere outside, unless you explicitly open it.

Reply:
Try instead to generate a deterministic token as here:

https://github.com/projectnessie/nessie/blob/7d71fe61ca6cee14f4c457ef4764f54b23d91957/helm/nessie/templates/deployment.yaml#L20

Reply:
Like this now?